packages/metastream-app/src/locale/ar.ts

```diff
@@ -1,103 +1,113 @@
   noticeAddedMedia: 'تم إضافة <Media id="{{mediaId}}">{{mediaTitle}}</Media>',
-  noticeMediaError: 'حدث خطأ أثناء الطلب {{url}}',
+  noticeMediaError: 'هنالك مشكلة عند طلب {{url}}',
   noticeNowPlaying:
-    'الآن يتم عرض  <Media id="{{mediaId}}">{{mediaTitle}}</Media> طلبت من <Username id="{{userId}}">{{username}}</Username>.',
-  noticeUserRequestJoin: '<Username id="{{userId}}">{{username}}</Username> يتطلب موافقة للإنضمام',
-  offline: 'غير متصل على الانترنت',
+    'الان يتم عرض  <Media id="{{mediaId}}">{{mediaTitle}}</Media> requested by <Username id="{{userId}}">{{username}}</Username>.',
+  noticeUserRequestJoin:
+    '<Username id="{{userId}}">{{username}}</Username> يطلب الانضمام',
+  offline: 'منقطع من الانترنت',
   ok: 'OK',
-  openInBrowser: 'فتح في المتصفح ',
+  openInBrowser: 'فتح في المتصفح',
   private: 'خاص',
-  profile: 'الملف شخصي',
+  profile: 'الملف الشخصي',
   public: 'عام',
   reload: 'إعادة التحميل',
   remove: 'إزالة',
   repeat: 'تكرار',
-  requestUrl: 'إضافة للبث',
+  retry: 'إعادة المحاولة ',
+  requestUrl: 'إضافة للجلسة',
   requestUrlPlaceholder: '',
-  requiresDJPermissions: 'يتطلب صلاحيات ال DJ',
-  requiresPortForward: 'يتطلب <1> إعداد ميناء شحن </1> من <3>{{port}}</3> لقبول الإتصال المباشر',
-  sessionMode: 'وضع البث',
-  sessionModePublicDescription: 'يسمح لأي شخص بالإنضمام',
-  sessionModePrivateDescription: 'يتطلب موافقة لللإنضمام.',
-  sessionModeOfflineDescription: 'لا يسمح لأي شخص بالإنضمام',
+  requiresDJPermissions: ' DJتتطلب صلاحيات ',
+  requiresPortForward:
+    'يتطلب <1> إعداد إعادة توجيه المنفذ </ 1> من <3> {{port}} </3> لقبول الاتصالات المباشرة',
+  sessionMode: 'وضع الجلسة',
+  sessionModePublicDescription: 'اي احد مسموح بالانضمام',
+  sessionModePrivateDescription: 'يتطلب موافق للانضمام',
+  sessionModeOfflineDescription: 'لا احد مسموح له الانضمام',
   settings: 'الإعدادات',
-  shareFriendCode: ' شارك أصدقائك الكود الذي في الأسفل لدعوتهم',
-  shareFriendCodeOther: 'شارك {{name}}’s أصدقائك الكود الذي في الأسفل لدعوتهم',
-  sharePublicIP: 'شارك <1>عنوان الIP الخاص بك</1> للسماح لأصدقائك بالإنضمام مباشرة',
-  shareSessionUrl: 'شارك URL البث الذي في الأسفل لدعوة الأصدقاء.',
-  startSession: 'بدء البث',
+  shareFriendCode: 'شارك اصدقائك هذا الرقم لدعوتهم',
+  shareFriendCodeOther: 'شارك{{name}} لدعوة اصدقائك',
+  sharePublicIP: 'شارك <1>IP</1> للسماح لأصدقائك بالاتصال مباشرة',
+  shareSessionUrl: 'شارة رابط الجلسة لدعوة أصدقائك',
+  startSession: 'بدء الجلسة',
   thirdPartyIntegrations: 'تكامل الجهات الخارجية',
   toggleDJ: 'منح DJ',
   unlimited: 'غير محدود',
-  unlockQueue: 'فتح قائمة الإنتظار',
-  username: 'إسم المستخدم ',
+  unlockQueue: 'فتح قائمة الانتظار',
+  username: 'اسم المستخدم ',
   users: 'المستخدمين',
-  userJoined: '<Username id="{{userId}}">{{username}}</Username> إنضم',
-  userLeft: '<Username id="{{userId}}">{{username}}</Username> غادر',
+  userJoined: '<المستخدم id="{{userId}}">{{username}}</Username> انضم',
+  userLeft: '<المستخدمid="{{userId}}">{{username}}</Username> غادر',
```

Comment:
The start of `userJoined` and `userLeft` should be `<Username id="`. Looks like it might have accidentally been translated.

Reply:
sry for that i'll try to be more careful about it